configure

```diff
@@ -399,14 +399,20 @@ fi
 
 # check if we are cross compiling
 checking 'whether we are cross compiling'
-if
-    tryexec
-then
-    checked "no"
+if [ -z "$cross_compiling" ]; then
```

Comment:
If this is expected to be given as an environment variable, it should be all-caps and listed in the list of `Influential environment variables`.